### DISCUSSION TITLE
Simplify Miller indices (or orientations) by setting maximum miller index

### DISCUSSION THREAD

**Discussion by SungwookChoi-4dev** [mtex-toolbox/mtex#1143] [Ask Anything] [answered]
Hi,

I'm now trying on determine Miller indices from orientations. 

Although "round2Miller(ori)" facilitate Miller index deduction from orientations, its maximum Miller index was 11 that is too complicated to further analysis. "maxindex"  options does not works for orientation variable. 

Is there any tips for simplifying orientations to Miller indices with typical maximum index regulation? 

Thanks a lot!

**Comment by kilir** [comment, marked as answer]
Hi,
it's `'maxIndex'` (or `'maxHKL'`) not `'maxindex'` 
If you had the spelling correct, maybe you need to explain what you mean by "does not work".

Cheers,
Rüdiger

**Reply by SungwookChoi-4dev** [reply]
Thanks for reply! 

"dose not work" means transform to Miller index always output Miller index with high index value (11 in below case) without regard to options. 
![image](https://user-images.githubusercontent.com/87750548/126452800-293d3044-4c50-4c5e-8d33-493d8469ea1f.png)

**Reply by SungwookChoi-4dev** [reply]
Thanks Rüdiger!

It seems that, one more rounding using round(m, 'maxHKL', num of index)

makes it possible simplifying Miller index. Thank you for advise!

 
![image](https://user-images.githubusercontent.com/87750548/126454972-2f6e0caa-15dc-45ea-8a85-261e090f0f13.png)
